static/bidder-info/consumable.yaml

```diff
@@ -1,4 +1,9 @@
 endpoint: "https://e.serverbid.com"
+extra_info: "{\"target_currency\": \"USD\"}"
```

Comment:
Why are you specifying extra info? The consumable adapter does not support any extra info.